apps/platform/src/components/ui/loading.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * Loading Component
+ * 
+ * This component creates a skeleton representation of a UI structure,
+ * followed by rendering actual data after loading. It utilizes utility
+ * functions (`cn` for classNames).
+ * Uses Skeleton component from  '@/components/ui/skeleton'
+ * Props:
+ * - className: Additional classNames to apply to the main container.
+ * 
+ * Features:
+ * - Generates 3 rows of skeleton content to mimic the "Key Manager" layout.
+ * 
+ * Uses:
+ * ```
+ * @/lib/utils - to create the structure of skeleton.
+ * ```
+ * Author: Nirajan1-droid
+ * Date: 2024-07-08
+ */
+import * as React from 'react';
+import { cn } from '@/lib/utils';  
+import { Skeleton } from '@/components/ui/skeleton'; 
+
+const Loading = React.forwardRef<HTMLDivElement>(() => {
+  return (
+    <div className="grid grid-cols-1 gap-4">
+      {/* Here, I have made 3 rows of skeleton to represent on the page. */}
+      {[...Array(3)].map((_, index) => (
+        <div
+          key={index}
+          className={cn(
+            'rounded-xl bg-white/5 px-5 py-4',
+            'data-[orientation=vertical]',

```

Comment:
No need to use `cn` utility here as this is only needed for classname that can conflict, change it to 
```css
className="data-[orientation=vertical] rounded-xl bg-white/5 px-5 py-4"
```